### DISCUSSION TITLE
Suggested docs improvement/clarification (query caching)

### DISCUSSION THREAD

**Discussion by adamcameron** [wheels-dev/wheels#1258] [General]
Hi
I'm reading this: https://guides.cfwheels.org/cfwheels-guides/handling-requests-with-controllers/caching#query-caching, specifically:

> [...] the cachedwithin attribute in the <cfquery> tag. The query caching in CFWheels is very similar to this.
> You can use query caching on all finder methods, and it looks like this:
> `users = model("user").findAll(where="name LIKE 'a%'", cache=10);`

The problem is that `cachedWithin` takes a timespan, whereas in your example yer passing `10`. `10` _what_?

I dug through the code (I am on 1.4.x btw, don't ask), and discovered the answer seems to be "it depends". 

Looking at https://github.com/cfwheels/cfwheels/blob/develop/wheels/global/internal.cfm#L212, it looks like there's an application-scoped(!) variable that this `$timeSpanForCache` method looks for, and it seems _that_ default's to `n` (https://github.com/cfwheels/cfwheels/blob/develop/wheels/events/onapplicationstart.cfm#L202), so the answer is "`10` means `10 minutes".

So I was wondering if it might be worth updating the docs to explain that the way CFWheels does it actually isn't terribly similar to how CFML does it natively (which would be with a parameter called `cachedWithin` that takes timespan, easy and already known to all CFML devs), but here's how it actually works. Summarising the intent of this `application.$wheels.cacheDatePart` variable, and how it relates to the value of the `cache` parameter there.

Cheers.

**Comment by chapmandu** [comment]
I find the API docs sometimes more useful than the guides. https://api.cfwheels.org/v1.4.5#findall

The cfwheels guides are now part of the repo (./guides) if wanted to contribute a PR.
